forge/test/models/onnx/audio/test_whisper_onnx.py

```
@@ -0,0 +1,109 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import torch
+import onnx
+from transformers import AutoProcessor, WhisperConfig, WhisperForConditionalGeneration
+
+import forge
+from forge.forge_property_utils import (
+    Framework,
+    ModelArch,
+    Source,
+    Task,
+    record_model_properties,
+)
+from forge.verify.verify import verify
+
+from test.utils import download_model
+
+
+class Wrapper(torch.nn.Module):
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+
+    def forward(self, input_features, decoder_input_ids):
+        inputs = {"input_features": input_features, "decoder_input_ids": decoder_input_ids}
+        output = self.model(**inputs)
+        return output.logits
+
+
+variants = [
+    "openai/whisper-tiny",
+    "openai/whisper-base",
+    "openai/whisper-small",
+    "openai/whisper-medium",
+    pytest.param(
+        "openai/whisper-large",
+        marks=[
+            pytest.mark.skip(reason="Fatal Python error: Aborted"),
```

Instead of skipping, can we raise error that is mathinc to current ones in PyTorch? I.e. to have better grouping in our analities. Feel free to sync with @meenakshiramanathan1, as when pushed this effort of skip removals :))

---

synced with @meenakshiramanathan1 . changed skip markers present in this PR to xfail marks